prep-javascript-objects/main.js

```diff
@@ -0,0 +1,18 @@
+var person = {
+  firstName: 'Caden',
+  lastName: 'Song',
+  hobby: 'Dancing'
+};
+console.log(person);
+
+var fullName = 'Caden ' + 'Song';
```

Comment:
Hi David, here you can get the first and last name from the person object using dot notation.
```js
var fullName = person.firstName + ' ' + person.lastName;
```

Reply:
![js object ](https://user-images.githubusercontent.com/90665895/134951376-8f1028bb-b444-4599-af3e-acbbbb01a598.png)